test/extensions/filters/udp/udp_proxy/udp_proxy_filter_test.cc

```diff
@@ -130,18 +179,21 @@ class UdpProxyFilterTest : public testing::Test {
 
   ~UdpProxyFilterTest() override { EXPECT_CALL(callbacks_.udp_listener_, onDestroy()); }
 
-  void setup(const std::string& yaml, bool has_cluster = true) {
+  void setup(const std::string& yaml, bool has_cluster = true, bool may_occur_exception = false) {
```

Comment:
may_occur_exception is never true in any test cases. Remove it?

Reply:
No, it is used in the ExitIpTransparentNoPlatformSupport test.
Please refer following code.
`EXPECT_THROW_WITH_REGEX(
      setup(config, true), EnvoyException,
      "The platform does not support either IP_TRANSPARENT or IPV6_TRANSPARENT. Or the envoy is "
      "not running with the CAP_NET_ADMIN capability.");`

Reply:
In that test, setup() is only supplied with 2 parameters. So may_occur_exception is still the default value

Reply:
Oh.... Sorry. I did not see that. I will fix it. Thanks your good point!!!